src/app/auth/register/components/recaptcha/recaptcha-validator.service.ts

```diff
@@ -14,7 +14,7 @@ export class RecaptchaValidator {
 
     validateToken(token: string) {
```

Comment:
Nit: Can this be moved to the user service?

Reply:
it was moved to the user service but i'm not able to test it because the Register page doesn't work, which is its only use.